proto/storage/active_component.proto

```diff
@@ -10,14 +10,18 @@ package storage;
 // Next available tag: 3
 message ActiveComponent {
     // base 64 encoded Deployment:ActiveComponent ids.
-    string id = 1;
+    string id            = 1 [(gogoproto.moretags) = 'search:"Image Sha,hidden"'];
+    string deployment_id = 3 [(gogoproto.moretags) = 'search:"Deployment ID,hidden"'];
+    string component_id  = 4 [(gogoproto.moretags) = 'search:"Component ID,hidden"'];
 
     // Represent a context of the active edge.
     message ActiveContext {
-        string container_name = 1;
-        string image_id       = 2 [(gogoproto.moretags) = "search:\"Image Sha,hidden,store\""];
+        string container_name = 1 [(gogoproto.moretags) = "search:\"Container Name,hidden\""];
+        string image_id       = 2 [(gogoproto.moretags) = "search:\"Image Sha,hidden\""];
     }
 
     // Map from container name to the active context of an edge.
-    map<string, ActiveContext> active_contexts = 2;
+    map<string, ActiveContext> DEPRECATED_active_contexts   = 2  [(gogoproto.moretags) = 'search:"-"'];
```

Comment:
Is this exposed on external APIs? Changelog?

Reply:
This is not exposed directly, and is only exposed as ActiveState